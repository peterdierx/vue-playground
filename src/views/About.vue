<template>
  <div class='box'>
    <h5 class='title is-5 has-text-primary'>About Vue Playground</h5>
    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
  </div>
</template>

<script>
  export default {
    name: 'About'
  }
</script>
