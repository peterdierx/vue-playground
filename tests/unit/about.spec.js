import { shallowMount } from '@vue/test-utils'
import About from '@/views/About'

describe( 'About Component', () => {
  
  it( 'renders about view', () => {
    const wrapper = shallowMount( About )
    expect( wrapper.element ).toMatchSnapshot()
  })
  
})
