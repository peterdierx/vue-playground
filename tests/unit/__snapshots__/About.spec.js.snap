// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Component renders about view 1`] = `
<div
  class="box"
>
  <h5
    class="title is-5 has-text-primary"
  >
    About Vue Playground
  </h5>
   
  <p>
    Just a sandbox to play with Vue, Vue Router, Vuex and Bulma.
  </p>
</div>
`;
